core/src/main/scala/io/finch/internal/ToEffect.scala

```diff
@@ -1,12 +1,13 @@
 package io.finch.internal
 
 import cats.effect.Effect
-import cats.~>
 import com.twitter.util.{Future => TwitterFuture, Return, Throw}
 import scala.concurrent.{Future => ScalaFuture}
 import scala.util.{Failure, Success}
 
-trait ToEffect[A[_], B[_]] extends ~>[A, B]
+trait ToEffect[F[_], G[_]] {
```

Comment:
Looks like we don't have to change this?

Reply:
Yep